packages/babel-preset-default/bin/index.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const builder = require( 'core-js-builder' );
+const { minify } = require( 'uglify-js' );
+const { writeFile } = require( 'fs' ).promises;
+
+builder( {
+	modules: [ 'es', 'web' ],
+	// core-js is extremely conservative in which polyfills to include.
+	// Knowing about tiny browser implementation bugs that anyone rarely cares about,
+	// we prevent some features from having the full polyfill included.
```

Comment:
I don't really understand the comment. What does core-js being conservative have to do with use excluding more things? And what tiny browser implementation bugs? To me the latter makes it sound like it reimplements everything?